meetings/2018-10-31.md

```diff
@@ -0,0 +1,92 @@
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2018-10-31
+
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=EK494IEUFOQ
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/618
+
+## Present
+
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Franziska Hinkelmann @fhinkel (TSC)
+* Jeremiah Senkpiel @Fishrock123 (TSC)
+* Joyee Cheung @joyeecheung (TSC)
+* Matteo Collina @mcollina (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Sakthipriyan Vairamani @thefourtheye (TSC)
+* Rich Trott @Trott (TSC)
+
+## Agenda
+
+### Announcements
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* New collaborator nominations
+  * Peter Marshall (https://github.com/nodejs/node/issues/23849)
+  * Ouyang Yadong (https://github.com/nodejs/node/issues/23955)
+  * Masashi Hirano (https://github.com/nodejs/node/issues/23850)
+
+### nodejs/node
+
+* fs: behaviour of readFile and writeFile with file descriptors [#23433](https://github.com/nodejs/node/issues/23433)  
+  * Vote has been suggested.
+  * Michael have we had complaints from end users
+    * Sakthipriyan - believe just raised by collaborators
```

Comment:
@jorangreef is not a collaborator, and originally raised  https://github.com/nodejs/node/issues/9671
@thisconnect and @bzoz from https://github.com/nodejs/node/issues/13572 are not collaborators

Reply:
@bzoz is a collaborator.

Reply:
@sam-github Ah, thanks for correcting my understanding.

Reply:
> behavior of readFile and writeFile with file descriptors

Thank you all for looking into this and taking action! It doesn't matter which direction you choose, everything is better than current situation. There only mistake would be to do nothing. :)